fex2/preconfigured/install/phoenix.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# MANDATORY HEADER: DO NOT MODIFY!
+if [ -z "${PROJ_ROOT}" ] ; then echo "Env. variable PROJ_ROOT must be set!" ; exit 1; fi
+source "${PROJ_ROOT}"/install/common.sh
+# END HEADER
+
+sudo apt-get install -y wget libc6-dev-i386
+
```

Comment:
Don't forget to download the sources. Example: https://github.com/tudinfse/fex/blob/fex-2/fex2/preconfigured/install/splash.sh